main.go

```diff
@@ -41,13 +43,26 @@ func main() {
 	}
 
 	myLinker := linker.New(myConfig.OSSIndexConfig)
-	count := myLinker.DoLink(args)
-	if count > 0 {
+	results := myLinker.DoLink(args)
+	if results.Count > 0 {
 		if context.ExitWithError() {
-			fmt.Fprintf(os.Stderr, "Error: Vulnerable dependencies found: %v\n", count)
-			os.Exit(count)
+			fmt.Fprintf(os.Stderr, "Error: Vulnerable dependencies found: %v\n", results.Count)
+			os.Exit(results.Count)
```

Comment:
This isn't totally related to this PR, but @ken-duck loves to exit with the number of vulnerable components. I'm a big fan of `os.Exit(1)` because it's a common code. I would argue that we SHOULD be using 1, because using the number can potentially overlap with actual special meaning codes: https://tldp.org/LDP/abs/html/exitcodes.html

It's one of those things that PROBABLY doesn't matter, but it's POSIX/Unix stuff so we may as well follow conventions. No need to hold up this PR, just a general observation!

Reply:
i agree